cas-server-core/src/main/java/org/jasig/cas/util/UniqueTicketIdGenerator.java

```diff
@@ -22,14 +22,13 @@
  * Interface that enables for pluggable unique ticket Ids strategies.
  *
  * @author Scott Battaglia
-
  * @since 3.0.0
- * <p>
- * This is a published and supported CAS Server 3 API.
- * </p>
  */
 public interface UniqueTicketIdGenerator {
-
+    /**
+     * Default instance for the ticket id generator.
+     */
+    UniqueTicketIdGenerator DEFAULT = new DefaultUniqueTicketIdGenerator();
```

Comment:
At the time, I couldnt  think of a better place to put it, but  now I have had time to sleep (on it!) maybe we could stuff the default instance into CASImpl itself.